_faqs/WillItWorkOnMyMachine.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Will BciPy work on my machine?
+categories: [FAQ]
+---
+
+BciPy is designed to install on the latest Windows (7, 10, 11), Linux (Ubuntu 22.04) and MacOS (Big Sur) operating systems. Other versions may work as well, but are not guaranteed. To see the updated list of supported python versions and operating systems visit the GitHub page [here](https://www.github.com/CAMBI-tech/BciPy). Please see the [Hardware and OS Configuration](https://bcipy.github.io/hardware-os-config/) page for additional information on hardware requirements for use in data collection.
```

Comment:
Fyi I'm developing on MacOS Monterey (OS 12) which is a later version than Big Sur